.changeset/silly-brooms-beg.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": patch
+---
+
+Fixed: `function-no-unknown` false positives for `running`
```

Comment:
```suggestion
Fixed: `function-no-unknown` false positives for `running()`
```

Reply:
I was matching the previous changesets.
i.e. 078d7f8, 4d32d3